src/Logic/logicGame.ts

```diff
@@ -0,0 +1,141 @@
+/* eslint-disable no-fallthrough */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-alert */
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+/* eslint-disable no-param-reassign */
```

Comment:
The same thing with comments.
Configure your eslint without comments or disable it at all.